source/js/components/petition/petition.jsx

```diff
@@ -107,11 +107,11 @@ class Petition extends Component {
 
     window.dataLayer = window.dataLayer || [];
     window.dataLayer.push({
-      'event' : 'form_submission',
```

Comment:
this was happening with prettier with daniel, I need to probably add some fields to our prettierrc instead of relying on the default configs

Reply:
I think the problem is also that `inv test` does not run a check with prettier, but rather performs the formatting. I think it would be easier to deal with if these steps are separated. `inv test` should only run checks that can fail, and e.g. use `inv format` to perform formatting operations.